test/test-mc-fle2-payload-iev-v2.c

```diff
@@ -591,6 +591,28 @@ static void test_fle2_iev_v2(_mongocrypt_tester_t *tester) {
     mongocrypt_destroy(crypt);
 }
 
+static void test_fle2_iev_v2_parse_invalid_input(_mongocrypt_tester_t *tester) {
+    mongocrypt_status_t *status = mongocrypt_status_new();
+    mc_FLE2IndexedEncryptedValueV2_t *iev = mc_FLE2IndexedEncryptedValueV2_new();
+
+    const size_t minValidEqualityLength = 1 + UUID_LEN + 1 + kMinServerEncryptedValueLen + kMetadataLen;
+    _mongocrypt_buffer_t input;
+
+    uint8_t data[minValidEqualityLength];
```

Comment:
This appears to cause a [build error on MSVC](https://spruce.mongodb.com/task/libmongocrypt_windows_test_build_and_test_and_upload_patch_9ed7c48c7034a93a743c149bfe0f3b4ecea206bb_687827e9f5bf4a00070e9f77_25_07_16_22_30_06/logs?execution=1):

```
test-mc-fle2-payload-iev-v2.c(601): error C2057: expected constant expression
```

Suggest dynamically allocating:
```c
uint8_t *data = (uint8_t *)bson_malloc0(minValidEqualityLength);
```

Followed by `bson_free (data)`.